notebooks/02_Fairing/Dockerfile.model

```diff
@@ -0,0 +1,9 @@
+#This container contains your model and any helper scripts specific to your model.
```

Comment:
Is it better to create a folder for this example? 

- Dockerfile.model
- model.py
- requirement.txt

Reply:
If we create a folder like `src`, it's better to also sort all of `02_Fairing`'s files, including `train.py`, `distributed_training.py`, etc.

Reply:
I mean you can create a  `kaniko` folder under `02_Fairing`, put these file there. These files will be used by notebook.